### PR TITLE
Upgrade to django 3

### PR DESCRIPTION
**Pull request by KipSigei**
- Upgrade to django v3.2.13
- Upgrade to use python 3.9

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -12,16 +12,15 @@
     url="https://github.com/onaio/ona-oidc",
     packages=find_packages(exclude=["tests"]),
     install_requires=[
-        "Django >= 2.2",
-        "djangorestframework >= 3.9",
+        "Django>=3.2.13,<4",
```
Comment (DavisRayM) [inline setup.py:15]: I wonder if we should switch to using a `setup.cfg` file instead ?

File: setup.py [hunk 2]
```diff
@@ -12,16 +12,15 @@
...
         "requests",
     ],
     classifiers=[
         "Programming Language :: Python",
-        "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.9",
         "Framework :: Django",
-        "Framework :: Django :: 2.2",
-        "Framework :: Django :: 3.0",
+        "Framework :: Django :: 3.2.13",
```
Comment (DavisRayM) [inline setup.py:24]:
```suggestion
        "Framework :: Django :: 3.2.13",
        "Framework :: Django :: 4.0.0",
```

File: setup.py [hunk 3]
```diff
@@ -12,16 +12,15 @@
...
-        "djangorestframework >= 3.9",
+        "Django>=3.2.13,<4",
+        "djangorestframework",
         "pyjwt[crypto]",
         "requests",
     ],
     classifiers=[
         "Programming Language :: Python",
-        "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.9",
```
Comment (DavisRayM) [inline setup.py:22]:
```suggestion
        "Programming Language :: Python :: 3.8",
        "Programming Language :: Python :: 3.9",
```

File: oidc/__init__.py [hunk 4]
```diff
@@ -2,6 +2,5 @@
 Main init file for oidc app
 """
 
-VERSION = (0, 0, 10)
+VERSION = (1, 1, 0)
```
Comment (DavisRayM) [inline oidc/__init__.py:5]:
```suggestion
VERSION = (1, 0, 0)
```

Comment (DavisRayM) [reply oidc/__init__.py:5]: Do we use this anywhere ? I believe it's safe to remove if we don't

Comment (KipSigei) [reply oidc/__init__.py:5]: Can be removed, it was initially used in setuptools

**Review by DavisRayM — APPROVED** [review]

**Pull request merged**